modules/options.js

```diff
@@ -230,9 +244,14 @@ export class StreamViewOptions {
 			scope: 'world',
 			config: true,
 			restricted: true,
-			requiresReload: true,
-			default: true,
-			type: Boolean,
+			requiresReload: false,
```

Comment:
Should we not reload to apply this change?

Reply:
I've been testing by importing my branch into a Forge hosted world, and it doesnt seem to need it. Changing the setting and pressing save will still toggle the chat on my other PC.